python/lsst/afw/geom/ellipses/ellipsesLib.py

```diff
@@ -1 +1,5 @@
-from __future__ import absolute_import
\ No newline at end of file
+from __future__ import absolute_import
+
+from ._baseCore import *
+from ._quadrupole import *
+from ._ellipse import *
```

Comment:
I think these are right and the wildcards should be used. They are forwarding the API into `ellipsesLib.py` to retain compatibility with the approach used by SWIG.